post-processor/amazon-import/post-processor.go

```diff
@@ -63,6 +64,10 @@ func (p *PostProcessor) Configure(raws ...interface{}) error {
 		p.config.S3Key = "packer-import-{{timestamp}}.ova"
 	}
 
+	if p.config.LicenseType == "" {
+		p.config.LicenseType = "BYOL"
```

Comment:
thanks! Can we change this then so we don't set a license type unless it's explicitly given?